src/eradiate/scenes/atmosphere/_molecular_atmosphere.py

```diff
@@ -399,6 +445,7 @@ def ussa_1976(
         cls,
         levels: pint.Quantity | None = None,
         concentrations: dict[str, pint.Quantity] | None = None,
+        wavelength_range: pint.Quantity = DEFAULT_WAVELENGTH_RANGE,
```

Comment:
This default is still mutable. The correct way to deal with that is to set it to an unmutable value, typically `None`, and generate the default in the function body. See [this page](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) for further detail.